internal/server/handlers.go

```diff
@@ -148,14 +150,15 @@ func (h *handler) loop() {
 			}
 		}
 
-		entries = h.swapResourceMaps(entries)
-
+		entries := h.swapEntries()
+		// TODO: have an admin UI that shows which clients are lagging the most
```

Comment:
Nit: you could open a github issue to track this

Reply:
I'm gonna remove that TODO altogether, it's definitely something that's more internal to what we want to have for the observer. `diderot` itself doesn't have the UI elements we use at LinkedIn, and we don't have to introduce that UI here, it can be added with an interceptor, so that TODO doesn't really have a place here